packages/amplify-graphql-api-construct-tests/src/__tests__/backends/amplify-table/canary/app.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
```

Comment:
Where are the backends for the other 3 strategies?

Related: we could use a shared backend app and write an IAmplifyGraphqlDefinition to the project directory from test code, as in:

- [test code](https://github.com/aws-amplify/amplify-category-api/commit/fd7f6fbc17c199331c4b04debaff69ea0424cd74#diff-89cc8302e4902cbe0102a8702c74965bda7e6b585f4508db0505c661cc4e5248R51-R58)
- [app.ts code](https://github.com/aws-amplify/amplify-category-api/commit/fd7f6fbc17c199331c4b04debaff69ea0424cd74#diff-0fa432347086e6bb7073d43f64322ccb461947c6886dca5a7095752eab4c4475R19-R34)

However, this isn't a shared test resource yet, so it's reasonable to publish this PR without using those utilities so we can start gathering canary data sooner

Reply:
The other 3 strategies re-use the backends we added for existing E2Es (`sql-models`, `base-cdk`) that serve the purpose for the state of canary right now. Once we evolve the schema in the canary tests, this kind of abstraction will be useful.